### ISSUE TITLE
Rendering fails in normalizing flows tutorial

### ISSUE THREAD

**Issue by fritzo** [pyro-ppl/pyro#2567]
Math is not rendering in @stefanwebb 's new tutorial
https://pyro.ai/examples/normalizing_flows_i.html

To reproduce, run `make tutorial` and view locally. 
Here are the warnings I see when running `make tutorial`:

<details>

```
/Users/fritzo/github/pyro-ppl/pyro/tutorial/source/normalizing_flows_i.ipynb:31: WARNING: Unexpected indentation.
/Users/fritzo/github/pyro-ppl/pyro/tutorial/source/normalizing_flows_i.ipynb:29: WARNING: Inline interpreted text or phrase reference start-string without end-string.
/Users/fritzo/github/pyro-ppl/pyro/tutorial/source/normalizing_flows_i.ipynb:34: WARNING: Block quote ends without a blank line; unexpected unindent.
/Users/fritzo/github/pyro-ppl/pyro/tutorial/source/normalizing_flows_i.ipynb:35: WARNING: Inline interpreted text or phrase reference start-string without end-string.
/Users/fritzo/github/pyro-ppl/pyro/tutorial/source/normalizing_flows_i.ipynb:37: WARNING: Definition list ends without a blank line; unexpected unindent.
/Users/fritzo/github/pyro-ppl/pyro/tutorial/source/normalizing_flows_i.ipynb:41: WARNING: Unexpected indentation.
/Users/fritzo/github/pyro-ppl/pyro/tutorial/source/normalizing_flows_i.ipynb:39: WARNING: Inline interpreted text or phrase reference start-string without end-string.
/Users/fritzo/github/pyro-ppl/pyro/tutorial/source/normalizing_flows_i.ipynb:44: WARNING: Block quote ends without a blank line; unexpected unindent.
/Users/fritzo/github/pyro-ppl/pyro/tutorial/source/normalizing_flows_i.ipynb:60: WARNING: Inline interpreted text or phrase reference start-string without end-string.
/Users/fritzo/github/pyro-ppl/pyro/tutorial/source/normalizing_flows_i.ipynb:104: WARNING: Inline interpreted text or phrase reference start-string without end-string.
/Users/fritzo/github/pyro-ppl/pyro/tutorial/source/normalizing_flows_i.ipynb:104: WARNING: Block quote ends without a blank line; unexpected unindent.
/Users/fritzo/github/pyro-ppl/pyro/tutorial/source/normalizing_flows_i.ipynb:156: WARNING: Inline interpreted text or phrase reference start-string without end-string.
/Users/fritzo/github/pyro-ppl/pyro/tutorial/source/normalizing_flows_i.ipynb:409: WARNING: Inline interpreted text or phrase reference start-string without end-string.
/Users/fritzo/github/pyro-ppl/pyro/tutorial/source/normalizing_flows_i.ipynb:407: WARNING: Block quote ends without a blank line; unexpected unindent.
/Users/fritzo/github/pyro-ppl/pyro/tutorial/source/normalizing_flows_i.ipynb:408: WARNING: Inline interpreted text or phrase reference start-string without end-string.
/Users/fritzo/github/pyro-ppl/pyro/tutorial/source/normalizing_flows_i.ipynb:410: WARNING: Definition list ends without a blank line; unexpected unindent.
/Users/fritzo/github/pyro-ppl/pyro/tutorial/source/normalizing_flows_i.ipynb:411: WARNING: Definition list ends without a blank line; unexpected unindent.
/Users/fritzo/github/pyro-ppl/pyro/tutorial/source/normalizing_flows_i.ipynb:417: WARNING: Inline interpreted text or phrase reference start-string without end-string.
/Users/fritzo/github/pyro-ppl/pyro/tutorial/source/normalizing_flows_i.ipynb:417: WARNING: Block quote ends without a blank line; unexpected unindent.
/Users/fritzo/github/pyro-ppl/pyro/tutorial/source/normalizing_flows_i.ipynb:432: WARNING: Unexpected indentation.
/Users/fritzo/github/pyro-ppl/pyro/tutorial/source/normalizing_flows_i.ipynb:430: WARNING: Inline interpreted text or phrase reference start-string without end-string.
/Users/fritzo/github/pyro-ppl/pyro/tutorial/source/normalizing_flows_i.ipynb:438: WARNING: Inline interpreted text or phrase reference start-string without end-string.
/Users/fritzo/github/pyro-ppl/pyro/tutorial/source/normalizing_flows_i.ipynb:440: WARNING: Definition list ends without a blank line; unexpected unindent.
/Users/fritzo/github/pyro-ppl/pyro/tutorial/source/normalizing_flows_i.ipynb:594: WARNING: Unexpected indentation.
/Users/fritzo/github/pyro-ppl/pyro/tutorial/source/normalizing_flows_i.ipynb:592: WARNING: Inline interpreted text or phrase reference start-string without end-string.
/Users/fritzo/github/pyro-ppl/pyro/tutorial/source/normalizing_flows_i.ipynb:595: WARNING: Block quote ends without a blank line; unexpected unindent.
/Users/fritzo/github/pyro-ppl/pyro/tutorial/source/normalizing_flows_i.ipynb:612: WARNING: Unexpected indentation.
/Users/fritzo/github/pyro-ppl/pyro/tutorial/source/normalizing_flows_i.ipynb:610: WARNING: Inline interpreted text or phrase reference start-string without end-string.
/Users/fritzo/github/pyro-ppl/pyro/tutorial/source/normalizing_flows_i.ipynb:249: WARNING: File not found: 'torch.nn.Module'
/Users/fritzo/github/pyro-ppl/pyro/tutorial/source/normalizing_flows_i.ipynb:432: WARNING: Unknown target name: "g".
/Users/fritzo/github/pyro-ppl/pyro/tutorial/source/normalizing_flows_i.ipynb:432: WARNING: Unknown target name: "h".
/Users/fritzo/github/pyro-ppl/pyro/tutorial/source/normalizing_flows_i.ipynb:437: WARNING: Unknown target name: "g".
/Users/fritzo/github/pyro-ppl/pyro/tutorial/source/normalizing_flows_i.ipynb:437: WARNING: Unknown target name: "h".
/Users/fritzo/github/pyro-ppl/pyro/tutorial/source/normalizing_flows_i.ipynb:592: WARNING: Unknown target name: "q".
```

</details>
